solidus_dev_support.gemspec

```diff
@@ -38,7 +40,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'puma', '~> 4.3'
   spec.add_dependency 'rspec-rails', '~> 4.0.0.beta3'
   spec.add_dependency 'rspec_junit_formatter'
-  spec.add_dependency 'rubocop', '~> 0.76.0'
+  spec.add_dependency 'rubocop', '~> 0.76'
```

Comment:
The reason this was locked is that the configuration format for `.rubocop.yml` changes pretty often in minor versions, and it's easy for the `.rubocop.yml` provided by solidus_dev_support to become outdated if we allow users to update the RuboCop version independently.

Reply:
Yeah, I've come to the same conclusion at my own expenses, I guess we'll need to find another way to allow for different upstream configurations and versions, ideally I'd like extensions to be able to update rubocop on their own if they're ahead of dev-support… anyway I'll close this for now, it was a draft/experiment after all.